src/easylink/pipeline_schema_constants/testing.py

```diff
@@ -401,3 +401,36 @@
 
 
 LOOPING_EP_STEP_SCHEMA_PARAMS = (LOOPING_EP_STEP_NODES, SINGLE_STEP_EDGES)
+
+
+EP_PARALLEL_STEP_NODES = [
+    InputStep(),
+    EmbarrassinglyParallelStep(
+        step=ParallelStep(
```

Comment:
Shouldn't the parallel step have multiple substeps?

Reply:
That gets defined in the pipeline.yaml - this is the schema

Reply:
in other words, the `step` that we are going to run embarrassingly parallel-y is a ParallelStep, and that ParallelStep in turn has a `template_step` (of just a basic Step in this case). It's in the pipeline yaml used in the test that the user says to have multiple via the [`parallel` configuration key](https://github.com/ihmeuw/easylink/blob/sbachmei/mic-5786-5981/embarrassingly-parallel-parallel-steps/tests/specifications/integration/embarrassingly_parallel/pipeline_parallel_step.yaml#L3).